src/main/java/org/terracotta/statistics/ConstantValueStatistic.java

```diff
@@ -40,6 +31,14 @@ private ConstantValueStatistic(StatisticType type, T value) {
     this.type = type;
   }
 
+  public static <T extends Serializable> ConstantValueStatistic<T> instance(StatisticType type, T value) {
```

Comment:
To have nice static imports, shouldn't we call this method something like `constant()`

Reply:
agree! I wanted to do that also. I can now add this in the commit about the refactoring :-)